technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -28,8 +28,8 @@
       <term name="issue" form="short">H.</term>
       <term name="translator" form="verb-short">deutsch von</term>
       <term name="presented at">gehalten auf</term>
-      <term name="no date" form="short">o.&#8239;J.</term>
-      <term name="et-al">u.&#8239;a.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="et-al">u. a.</term>
```

Comment:
@jmiba, if you can help it, can you keep special characters like the one above escaped when making future edits? It might be enough to always start with the latest version from our repository, or, if you already did that, it might be a setting in your text editor.